src/main/java/ch/wisv/areafiftylan/extras/consumption/model/ConsumptionMap.java

```diff
@@ -37,7 +38,7 @@
 
     @NonNull
     @ManyToMany(fetch = FetchType.EAGER)
-    private Collection<Consumption> consumptionsMade;
+    private List<Consumption> consumptionsMade;
```

Comment:
Hibernate 5.0 doesn't like initializing 2 collections at once. By changing this to more specific implementations of a Collection, it was able to do so. Not exactly sure what the internal difference is, but  considering the nature of the collections, this doesn't hurt